swarm/CHANGELOG.md

```diff
@@ -1,8 +1,72 @@
 # 0.38.0 [unreleased]
 
-- Update dial address concurrency factor to `8`, thus dialing up to 8 addresses concurrently for a single connection attempt. See `Swarm::dial_concurrency_factor` and [PR 2741].
+- Remove `NetworkBehaviourEventProcess`. When deriving `NetworkBehaviour` on a custom `struct` users
+  should either bring their own `OutEvent` via `#[behaviour(out_event = "MyBehaviourEvent")]` or,
+  when not specified, have the derive macro generate one for the user.
 
-- Update to `libp2p-core` `v0.35.0`.
+  See [`NetworkBehaviour`
+  documentation](https://docs.rs/libp2p/latest/libp2p/swarm/trait.NetworkBehaviour.html) for
+  details.
+
```

Comment:
Was the PR not linked on purpose (same for the changelog below)?

Reply:
:+1: linked with b5cd98da